bin/zhtautau/runZHTauTauAnalysis.cc

```diff
@@ -1308,7 +1428,7 @@ int main(int argc, char* argv[])
 
 
         //SIGNAL ANALYSIS Z+2Leptons  (no systematics taken into account here)
-        if(passZmass && (int)selLeptons.size()>=4){  //Request at least 4 leptons
+        if(passZmass && passZpt && (int)selLeptons.size()>=4){  //Request at least 4 leptons
```

Comment:
The ptZ is not well modelled in NLO MC samples. The cut has been introduced to check whether it improves the Data/MC agreement